cluster/create.go

```diff
@@ -472,13 +498,18 @@ func createVirtualNetwork(ctx context.Context, subscriptionID, resourceGroup, lo
 }
 
 // createPostgresFlexibleServer provisions an Azure PostgreSQL Flexible Server matching the Bicep module configuration
-func createPostgresFlexibleServer(ctx context.Context, subscriptionID, resourceGroup, location, vnetNamePrefix, adminUsername, adminPassword, clusterHash string, cred *azidentity.DefaultAzureCredential) error {
+func createPostgresFlexibleServer(ctx context.Context, subscriptionID, resourceGroup, location, vnetNamePrefix, adminUsername, adminPassword, clusterHash, postgresSKU string, cred *azidentity.DefaultAzureCredential) error {
 	serverName := strings.ToLower(vnetNamePrefix + "pg" + clusterHash)
 	serversClient, err := armpostgresqlflexibleservers.NewServersClient(subscriptionID, cred, nil)
 	if err != nil {
 		return fmt.Errorf("failed to create postgres servers client: %w", err)
 	}
 
+	// Use default SKU if none provided
+	if postgresSKU == "" {
+		postgresSKU = "Standard_D2s_v3"
```

Comment:
This should come in via the default value of the flag to be consistent with the other options.